Duplicati/Library/Utility/TempFolder.cs

```diff
@@ -29,7 +29,7 @@ namespace Duplicati.Library.Utility
     /// </summary>
     public class TempFolder : IDisposable
     {
-        private string m_folder;
+        private string? m_folder;

```

Comment:
This should be `readonly` and a `m_isDisposed` flag should be added.

Reply:
Not sure where to add the m_isDisposed flag?